src/ui/component-repository/components/list/ComponentList.js

```diff
@@ -46,7 +46,7 @@ class ComponentList extends Component {
       perPage,
       perPageOptions: [5, 10, 15, 25, 50],
     };
-
+    console.log(componentRepositoryComponents);
```

Comment:
Hey @gReis89 ,
Could you please remove this console.log if not necessary?
Thanks

Reply:
done, thanks!